daisy/disk.go

```diff
@@ -145,6 +145,7 @@ func (d *Disk) validate(ctx context.Context, s *Step) DError {
 }
 
 type diskAttachment struct {
+	diskName           string
```

Comment:
this field is added so that we can find disk name by device name from attachments